wordpress-mobile/WordPress-Android
# Issue/7140 add epilogue email

### Fix
Add epilogue for email as described in https://github.com/wordpress-mobile/WordPress-Android/issues/7140.

### Test
0. Clear app data.
1. Tap ***Sign Up*** button.
2. Tap ***Sign Up with Email*** button.
3. Enter email address _not_ associated with WordPress.com account.
4. Notice ***Signup Magic Link*** screen.
5. Check email inbox of address used in Step 3.
6. Notice signup email from WordPress.com.
7. Tap ***Sign up to WordPress.com*** button in email.
8. Notice  ***Epilogue for Email*** screen.
9. Tap avatar in header.
10. Notice add avatar interface.
11. Notice avatar is updated once adding avatar is finished.
12. Tap ***Continue*** button.
  a. If display name, username, or password were changed, notice progress dialog.
  b. If display name, username, or password were not changed, notice no progress dialog.

#### Note
Some methods (i.e. [`startCropActivity`](https://github.com/wordpress-mobile/WordPress-Android/blob/issue/7140-add-epilogue-email/WordPress/src/main/java/org/wordpress/android/ui/accounts/signup/SignupEpilogueFragment.java#L550-L566) and [`startGravatarUpload`](https://github.com/wordpress-mobile/WordPress-Android/blob/issue/7140-add-epilogue-email/WordPress/src/main/java/org/wordpress/android/ui/accounts/signup/SignupEpilogueFragment.java#L568-L600)) were taken from [`MeFragment`](https://github.com/wordpress-mobile/WordPress-Android/blob/issue/7140-add-epilogue-email/WordPress/src/main/java/org/wordpress/android/ui/main/MeFragment.java#L412-L458) to handle uploading an avatar image.  We should consider moving these methods to a utility class or library to eliminate duplicated code.  We could also move methods like [`createDisplayNameFromEmail`](https://github.com/wordpress-mobile/WordPress-Android/blob/issue/7140-add-epilogue-email/WordPress/src/main/java/org/wordpress/android/ui/accounts/signup/SignupEpilogueFragment.java#L472-L490) and [`createUsernameFromEmail`](https://github.com/wordpress-mobile/WordPress-Android/blob/issue/7140-add-epilogue-email/WordPress/src/main/java/org/wordpress/android/ui/accounts/signup/SignupEpilogueFragment.java#L492-L515) to a utility class or library for use elsewhere.
